ReactAndroid/src/main/java/com/facebook/react/modules/datepicker/BUCK

```diff
@@ -10,6 +10,7 @@ android_library(
     react_native_target('java/com/facebook/react/bridge:bridge'),
     react_native_target('java/com/facebook/react/common:common'),
     react_native_target('java/com/facebook/react/module/annotations:annotations'),
+    react_native_target('res:shell'),
```

Comment:
Actually i'm not sure it's right or not add  `res:shell` dependency here, but i guess it's a bad smell...